docs/guides/add-exposure-tests.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Add exposure validation tests"
+---
+
+After you [install the dbt package](/quickstart#install-the-dbt-package), you can add Elementary exposure validation tests.
+
+## Exposure validation dbt tests
+
+Elementary dbt package includes **exposure validation tests, implemented as [dbt tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests)**.
+These tests can detect missing columns and column data types for defined exposures.
+
+## Configure your elementary exposure validation tests
+
+<Tip>
+  You can read up about exposures and how to add them at [the dbt docs
+  site](https://docs.getdbt.com/docs/build/exposures)
+</Tip>
+
+Within your `models` directory, add a file called `exposures.yml`
+
+```yml
+exposures:
+  - name: customers
+    label: Customer Dashboard
+    type: dashboard
+    maturity: high
+    url: https://your.bi.tool/dashboards/1
+    description: >
+      Shows customer growth
+
+    depends_on:
+      - ref('customers')
+
+    owner:
+      name: Callum McData
+      email: data@jaffleshop.com
+    meta:
+      columns:
+        - name: "customer_id"
+```
+
+<Note>You can optionally also specify the column data type (`data_type`)</Note>
+
+```yml
+- name: returned_orders
+  label: Returned Orders
+  type: dashboard
+  maturity: high
+  url: https://your.bi.tool/dashboards/2
+  description: >
+    Returned orders over time
+
+  depends_on:
+    - ref('returned_orders')
+
+  owner:
+    name: Callum McData
+    email: data@jaffleshop.com
+  meta:
+    columns:
+      - name: "order_id"
+        data_type: "numeric"
+```
+
+Then in your module schema, add the elementary exposure tests:
+
+```yml
+  ...
+  - name: returned_orders
+    description: This table contains all of the returned orders
+    config:
+      tags: ["finance"]
+
+    tests:
+      - elementary.volume_anomalies:
+          tags: ["table_anomalies"]
+          timestamp_column: "order_date"
+      - exposure_schema_validity:
+          tags: [elementary]
+
+```
+
+We recommend adding a tag to the tests so you could execute these in a dedicated run using the selection parameter `--select tag:elementary`.
```

Comment:
do we have this recommendation in all the test types?

Reply:
saw it in https://github.com/elementary-data/elementary/blob/ele-1703-create-generic-test-for-exposure-stability/docs/guides/add-elementary-tests.mdx#L234